addon/content/es-modules/utils.js

```diff
@@ -2,25 +2,85 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-/*
- * This is a temporary wrapper. Privileged code doesn't allow es-modules, but
- * eventually all components should be exported as es-modules. This code
- * "re-exports" globals as es-modules, which enables them to be imported
- * (e.g. for tests) or used as globals.
+/**
+ * Take a name and extract initials from it.
+ * If `name` is an email address, get the part before the @.
+ * Then, capitalize the first letter of the first and last word (or the first
+ * two letters of the first word if only one exists).
  *
- * When the switch to a WebExtension is done, the actual code should be migrated
- * here and the global variable workarounds should be removed.
+ * @param {string} name
+ * @returns {string}
  */
+export function getInitials(name) {
```

Comment:
Do we want to do this function in the UI, or when we generate the contact? If we did it when generating the contact, that would potentially avoid having to do it frequently, but at the same time, it is a UI type function.

What I'm thinking about, is we could potentially merge this file into ContactManager, or maybe separate the two functions into the places where they are used, rather than having a separate module.

Reply:
It was originally part of `ContactManager`, I believe, but I split it out for testing. I don't think it's an expensive function, and even running it 1000 times per display probably doesn't have much of an impact.

Reply:
That's fine, but I think it might still be worth splitting the file, so that we're only importing the functions necessary and have a clearer distinction between background and content code.